tensorflow/lite/micro/tools/ci_build/test_apollo3.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+#
+# Tests the microcontroller code using native x86 execution.
+
+set -e
+
+# SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+# ROOT_DIR=${SCRIPT_DIR}/../../../../..
+# cd "${ROOT_DIR}"
+
+# source tensorflow/lite/micro/tools/ci_build/helper_functions.sh
+
+# define a method to add apollo3 targets
+TARGETS=''
+add_target () {
+  TARGETS+=$1
+  TARGETS+=' '
+}
+
+# add all apollo3 targets
+add_target apollo3evb
+add_target sparkfun_edge
```

Comment:
also, let's start by only having ```TARGETS=sparkfun_edge``` (so that the behavior matches what we currently have).

In general, I want to make sure that the CI does not take too much time (since it is run as part of *every* pull request on tensorflow).

We can add apollo3evb to the CI as a separate PR. if we feel like the additional time is justified. We can also investigate having a nightly build with a build badge instead.